Tests/CodegenCLITests/Commands/InitializeTests.swift

```diff
@@ -191,7 +338,12 @@ class InitializeTests: XCTestCase {
 
       expect(actualPath).to(equal(expectedPath))
       expect(data?.asString).to(equal(
-        ApolloCodegenConfiguration.minimalJSON(schemaName: "MockSchema")))
+        ApolloCodegenConfiguration.minimalJSON(
```

Comment:
These tests are testing that you are calling `minimalJSON` with the correct parameters given the arguments you are parsing. But is there anything testing that `minimalJSON` is actually correct?

Reply:
This test at line 341 is to validate that the correct custom path is used when creating the initialization file and that the data to be written matches what is expected. Tests specifically for minimalJSON 'correctness' are down from line 478 onwards.

`minimalJSON` is exactly that, the minimal amount of JSON (`String`) to be able to decode an `ApolloCodegenConfiguration` instance and the rest are filled in by the defaults we have in the `ApolloCodegenConfiguration` decoding functions.

The tests are premised on:
* successful decoding proves the JSON is correct and valid (not malformed) otherwise `JSONDecoder().decode` would fail. This would also fail if a required parameter were added to `ApolloCodegenConfiguration` but not added to `minimalJSON`.
* each test is testing a value fed into `minimalJSON` to validate the logic for `supportCocoaPods` and `targetName`.

Reply:
> successful decoding proves the JSON is correct and valid (not malformed) otherwise JSONDecoder().decode would fail.

This is what I was looking for. Didn't see that we were actually decoding the JSON in the tests. Thanks!